xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -271,6 +271,7 @@ void CGUIWindowVideoBase::OnItemInfo(CFileItem* pItem, ADDON::ScraperPtr& scrape
     Refresh();
     m_viewControl.SetSelectedItem(itemNumber);
   }
+  return modified;
```

Comment:
I just ran it in the debugger and if I simply open and close the video info dialog `modified` is `false` which results in `OnItemInfo()` to return `false` which results in the video being automatically played. That's not the behaviour we want right?